src/components/SQFormDialog/SQFormDialogInner.js

```diff
@@ -46,7 +46,7 @@ const useActionsStyles = makeStyles({
 });
 const useDialogContentStyles = makeStyles({
   root: {
-    overflowY: 'visible',
+    maxHeight: '75vh',
```

Comment:
A question, why was it necessary to set a max height here?

Reply:
I bet it was because without a set height we don't have a scroll area to scroll within.  That is why it likely just scrolled the full parent container.  This is setting a height on the child to make that the scrollable area.  `75vh` not sure how that was calculated though.

Reply:
If the maxHeight is not set, the dialog will simply overflow the page and you will simply have to scroll down the page to see the end of the modal.  See the attached screenshot.

![sqFormDialog maxHeight](https://user-images.githubusercontent.com/16596855/141014228-e7adcaae-fa4f-4102-be99-e19a7c8503c3.png)

Reply:
Are you guessing that 25vh is how much room the header and footer take up?  75vh sounds mysterious.  How did you get that number?  I would also test the scenario that if we don't have enough fields for a scrollbar to show does it still look good.  A loom showing everything you test is best so that there aren't any questions about what was tested.

Reply:
Setting dialog conent dimensions based on the view port doesn't seem like the right choice. Ultimately would it be better to set the dialog to be a `#vh` tall and then the content be `#%` of that? While the footer and header will be another `#%`?

As the view port gets smaller the content is always utilize 75% of the view port while the dialog itself may be taking up a smaller amount.

Reply:
Attached is a video of how the dialog looks without any scrollbar needed.  It does not always take up 75% of the browser's viewport since I set this as maxHeight not height.  I can certainly set it to your suggestion @20BBrown14 where I explicitly set the entire dialog, than the content, header, and footer.  Let me know...

https://www.loom.com/share/53ace8a1fbb74bad93202fa02bc7c71d

Reply:
You're right, sorry. I misread the style rule. I think setting the maxHeight to be relative to the dialog itself instead of the view port would make more sense though. Hm. SQForm doesn't have a lot of styling decisions made quite yet. I'm definitely interested in getting more input, though.

Reply:
No worries.  I'll just wait for others to give their opinions so we can come to a consensus.